server/container/guice/mailrepository-cassandra/src/main/java/org/apache/james/modules/mailrepository/CassandraMailRepositoryModule.java

```diff
@@ -64,4 +68,12 @@ protected void configure() {
             .addBinding().to(MailRepositoryBlobReferenceSource.class);
     }
 
+    @ProvidesIntoSet()
```

Comment:
Using Multibinders directly in the configure method would allow not calling manually the constructor. 

More resilient to changes, less boiler plate...

Reply:
I'm not very familiar with the multibinder api, I tried it (for this module) in a08b9eff45bc2da5ad649dd8a5cbaeb761160d78